### PR TITLE
Project-Auth- Sara-Amanda

### PR DESCRIPTION
**Pull request by ssundblom**
Link to my deployed project:
https://auth-sara-amanda.netlify.app/#

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,9 +1,44 @@
...
+  flex-direction: column;
+`
+
+const Switch = styled.a`
+  text-align: center;
+  margin: auto;
+  background-color: inherit;
+  text-decoration: none;
+  color: #454545;
+`
```
Comment (KarolCislowski) [inline frontend/src/App.js:27]: nice usage of the styled-components 👍🏻

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Please change this:

1. User data should be stored in local storage to prevent logging out during page refresh,
2. Secret date suppose to be a separate page, not just a component under the form,
3. When your user logs out, then secret should be automatically deleted from the view.

**Comment by ssundblom** [comment]
Thank you for your valuable feedback! 
We added local storage- but we can't seem to find it in the requirements anywhere? As well as the separate page your talking about (the component under the form is our solution to try out the authentication of the user - showing a 403 if not properly signed in) 
Please point out in the blue requirements where we are lacking - have we misinterpreted something? 
We would off course love to improve, but due to time limitation it's hard for us to perfect this project.

**Comment by KarolCislowski** [comment]
> Then, on the frontend side of things, you'll need to build up a registration form that POSTs to your API. You'll need to store the access token you get back in the browser **using local storage**, and then use that token when making other requests to your API.

> Your frontend needs:
> 
> - A registration form.
> - A sign-in form.
> - **A page** to show the authenticated content from the API.
> - A 'sign out' button that removes the saved access token and redirects the user to the login form.


Instructions how it needs to be done, are at the beginning of the documentation. 😺 the lists of requirements are only short summaries and should be used as a checklist if you didn't forgot about anything. 😸

**Review by KarolCislowski — APPROVED** [review]
You should change one small thing - after logging out your 🥳 cat image should be no longer visible. It is a small fix, and I trust you will do it even when I Approved this PR ❤️ 

Good lock with big final projects! 🌈

**Comment by ssundblom** [comment]
Ah okey okey! Thank you for clearing that out :) And thank you for the
confidence :) Will fix asap!

WoopWoop! 🥳🥳

Den sön 24 jan. 2021 kl 15:40 skrev Karol Cislowski <
notifications@github.com>:

**Pull request closed without merging**